### ISSUE TITLE
Modify admin box connectable query

### ISSUE THREAD

**Issue by saposyprincesas** [scribu/wp-posts-to-posts#118]
Hi!

I have three CPTs: A, B, C

A has some B connected. C can also have some B connected, but only those already connected to A.

I think this can't be achived through admin box filters. I could add my own meta box, but I'm not sure how to extract all the functionality from box.php (and don't want to mess with all the ajax code). And of course I wouldn't like to modify the plugin core...
What would you recommend me to do? Is it safe to get out from box-factory.php the '$box = new P2P_Box( $box_args, $directed )' code? Is there an easier way to achive this?

Basically,  I need all admin box functionality, but need to change the connectable query.

Thank you!
